CRM/Event/Form/Participant.php

```diff
@@ -1546,7 +1546,7 @@ public function submit($params) {
         );
         $prefixValue = Civi::settings()->get('contribution_invoice_settings');
         $invoicing = $prefixValue['invoicing'] ?? NULL;
-        if (!empty($taxAmt) && (isset($invoicing) && isset($prefixValue['is_email_pdf']))) {
+        if (!empty($taxAmt) && (isset($invoicing) && Civi::settings()->get('invoice_is_email_pdf'))) {
```

Comment:
Is it possible to replace this condition with only `if (Civi::settings()->get('invoice_is_email_pdf')` ? I realize it might be a small change in behaviour, but can't we enable PDF invoices without taxes?

Reply:
@mlutfy good spotting - I'll change to that. In the UI it's hidden if invoicing is disabled but I think it's more logical to stand alone  - I think that change has been made in other places

Reply:
@mlutfy updated - I fixed all the places